addon/errors.js

```diff
@@ -1,11 +1,14 @@
 import Ember from 'ember';
 
-var get = Ember.get;
-var set = Ember.set;
+const { A } = Ember;
 
-export default Ember.Object.extend({
-  unknownProperty: function(property) {
-    set(this, property, Ember.A());
+const EmberObject = Ember.Object;
```

Comment:
The syntax is correct, but we cannot use destructuring with get and set at the moment (due to a bug in esprima used by JSCS).